src/main/java/com/epam/page/object/generator/builder/JavaClassBuilder.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.page.object.generator.builder;
+
+import static com.epam.page.object.generator.util.StringUtils.firstLetterUp;
+import static com.epam.page.object.generator.util.StringUtils.splitCamelCase;
+
+import com.epam.jdi.uitests.web.selenium.elements.composite.Form;
+import com.epam.jdi.uitests.web.selenium.elements.composite.Section;
+import com.epam.page.object.generator.adapter.classbuildable.FormClassBuildable;
+import com.epam.page.object.generator.adapter.javaclass.IJavaClass;
+import com.epam.page.object.generator.adapter.filed.IJavaField;
+import com.epam.page.object.generator.adapter.annotation.IJavaAnnotation;
+import com.epam.page.object.generator.adapter.javaclass.JavaClass;
+import com.epam.page.object.generator.adapter.classbuildable.PageClassBuildable;
+import com.epam.page.object.generator.adapter.classbuildable.SiteClassBuildable;
+import com.epam.page.object.generator.model.searchrule.FormSearchRule;
+import com.epam.page.object.generator.model.webgroup.FormWebElementGroup;
+import com.epam.page.object.generator.util.SearchRuleType;
+import java.util.List;
+import javax.lang.model.element.Modifier;
+
+public class JavaClassBuilder {
+
+    private String packageName;
+
+    public JavaClassBuilder(String packageName) {
+        this.packageName = packageName;
+    }
+
+    public IJavaClass visit(SiteClassBuildable siteClassBuildable) {
+        String classPackageName = packageName + ".site";
```

Comment:
I guess it is better to remove all unchanged values from methods and use them as constants. For example, private static final String SITE_EXTENSION = ".site"

Reply:
Done